pyaedt/desktop.py

```diff
@@ -1573,3 +1577,104 @@ def install(package_path, package_name=None):
         if aedt_version >= "2023.2":
             write_toolkit_config(os.path.join(toolkit_dir, product), lib_dir, toolkit_name, toolkit=toolkit)
         self.logger.info("{} toolkit installed.".format(toolkit_name))
+
+    @pyaedt_function_handler()
+    def submit_job(
+        self,
+        project_file,
+        clustername,
+        aedt_full_exe_path=None,
+        numnodes=1,
+        numcores=32,
+        wait_for_license=True,
+        setting_file=None,
+    ):  # pragma: no cover
+        """Submit a job to be solved on a cluster.
+
+        Parameters
+        ----------
+        project : str
+            Full path to project.
+        clustername : str
+            Name of the cluster to submit the job to.
+        aedt_full_exe_path : str, optional
+            Full path to the AEDT executable file. The default is ``None``, in which
+            case ``"/clustername/AnsysEM/AnsysEM2x.x/Win64/ansysedt.exe"`` is used.
+        numnodes : int, optional
+            Number of nodes. The default is ``1``.
+        numcores : int, optional
+            Number of cores. The default is ``32``.
+        wait_for_license : bool, optional
+             Whether to wait for the license to be validated. The default is ``True``.
+        setting_file : str, optional
+            Name of the file to use as a template. The default value is ``None``.
+
+        Returns
+        -------
+        type
+            ID of the job.
+
+        References
+        ----------
+
+        >>> oDesktop.SubmitJob
+        """
+
+        project_path = os.path.dirname(project_file)
+        project_name = os.path.basename(project_file).split(".")[0]
+        if not aedt_full_exe_path:
+            version = self.odesktop.GetVersion()[2:6]
+            if version >= "22.2":
+                version_name = "v" + version.replace(".", "")
+            else:
+                version_name = "AnsysEM" + version
+            if os.path.exists(r"\\" + clustername + r"\AnsysEM\{}\Win64\ansysedt.exe".format(version_name)):
+                aedt_full_exe_path = (
+                    r"\\\\\\\\" + clustername + r"\\\\AnsysEM\\\\{}\\\\Win64\\\\ansysedt.exe".format(version_name)
+                )
+            elif os.path.exists(r"\\" + clustername + r"\AnsysEM\{}\Linux64\ansysedt".format(version_name)):
+                aedt_full_exe_path = (
+                    r"\\\\\\\\" + clustername + r"\\\\AnsysEM\\\\{}\\\\Linux64\\\\ansysedt".format(version_name)
+                )
+            else:
+                self.logger.error("AEDT shared path does not exist. Please provide a full path.")
```

Comment:
```suggestion
                self.logger.error("AEDT shared path does not exist. Provide a full path.")
```
Matching the error message in line 1644. No need to use "please" in technical documentation.